app/helpers/application_helper/button/vm_snapshot_revert.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationHelper::Button::VmSnapshotRevert < ApplicationHelper::Button::Basic
```

Comment:
@tzumainn just curious if we should create a generic ApplicationHelper::Button::VmSnapshot class instead of specific one for just to handle a single button, and pass in feature so it can be used for other buttons if need arises.